src/carousel/carousel.ts

```diff
@@ -83,7 +84,10 @@ export class NgbSlide {
     <ol class="carousel-indicators" [class.visually-hidden]="!showNavigationIndicators" role="tablist">
       <li *ngFor="let slide of slides" [class.active]="slide.id === activeId"
           role="tab" [attr.aria-labelledby]="'slide-' + slide.id" [attr.aria-controls]="'slide-' + slide.id"
+          [attr.data-bs-target]="id"
```

Comment:
If you want to do as bootstrap, you should add a `#` before the id.
cf https://getbootstrap.com/docs/5.1/components/carousel/#with-indicators

But actually, I don't understand the point of adding an id at all. You could simply add the `data-bs-target` attribute with any value (or even without any value) for the CSS to be properly applied Do we actually need an id?